bemuse/src/music-previewer/index.js

```diff
@@ -187,12 +181,8 @@ function createMusicPreviewer() {
       loaded: false,
       play() {
         if (!played) {
-          try {
-            audio.play()
-            played = true
-          } catch (e) {
-            console.warn('Cannot play', audio.src)
-          }
+          audio.play().catch(() => console.warn('Cannot play', audio.src))
+          played = true
```

Comment:
We only want to set `played = true` when `audio.play()` is successful. By putting it outside the try-catch, `played = true` would be set regardless of whether the playing of the audio is successful or not.

```suggestion
          new Promise(r => r(audio.play()))
            .then(() => {
              played = true
            })
            .catch(() => console.warn('Cannot play', audio.src))
```

Reply:
Fix in 0682fd3c. Sorry for lack of investigation.